src/tools/histogram_dump.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+# coding: utf-8
+#
+# Ceph - scalable distributed file system
+#
+# Copyright (C) 2017 OVH
+#
+# This is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public
+# License version 2, as published by the Free Software
+# Foundation.  See file COPYING.
+#
+
+import json
+import subprocess
+import time
+import os
+import argparse
+
+
+def shorten(val):
+    if type(val) is str:
+        return val
+    for u in ((3, ''), (6, 'k'), (9, 'M'), (12, 'G'), (15, 'T')):
```

Comment:
Lower-case intentionally since it's 10^3, not 2^10 (as per https://en.wikipedia.org/wiki/K_(disambiguation)#Unit) - this tool doesn't really take into account axis units, always threats as raw integer values - seems like a good future improvement though.